crates/miden-tx/src/host/link_map.rs

```diff
@@ -42,13 +44,8 @@ impl<'process> LinkMap<'process> {
     /// Expected operand stack state before: [map_ptr, KEY, NEW_VALUE]
     /// Advice stack state after: [set_operation, entry_ptr]
     pub fn handle_set_event(process: &ProcessState<'_>) -> Result<Vec<AdviceMutation>, EventError> {
-        let map_ptr = process.get_stack_item(0);
-        let map_key = Word::from([
-            process.get_stack_item(4),
-            process.get_stack_item(3),
-            process.get_stack_item(2),
-            process.get_stack_item(1),
-        ]);
+        let map_ptr = process.get_stack_item(0 + 1);
+        let map_key = process.get_stack_word_be(1 + 1);
```

Comment:
We can probably write 1, 2 here

Reply:
No strong opinion, do we want to retain the explicitness of events or fold it in?

Reply:
I'd say in general if the docstring describes the stack with `event_id` then we can remove the offset